spec/system/casework/viewing_an_application/application_details/partner_details_spec.rb

```diff
@@ -124,8 +123,7 @@
 
         it "displays the partner's address details" do
           expect(summary_card('Partner details')).to have_rows(
-            'Lives at same address as client?', 'No',
-            'Home address', ''
+            'Lives at same address as client?', 'No'
```

Comment:
If they don't live at the same address as the client shouldn't the home address show? or is an address not stubbed?

Reply:
The home address is nil in this context, so the `Home address` row will not be visible. There's a test just above this one that tests the scenario you describe.